specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4594,6 +4618,56 @@
       },
       "description": "Resource provider information."
     },
+    "ResourceProviderAuthorization": {
+      "properties": {
+        "applicationId": {
+          "type": "string",
+          "description": "The application id of resource provider."
+        },
+        "roleDefinitionId": {
+          "type": "string",
+          "description": "The role definition id."
+        },
+        "managedByRoleDefinitionId": {
+          "type": "string",
+          "description": "The managed by role definition id."
+        },
+        "managedByAuthorization": {
+          "$ref": "#/definitions/ResourceProviderManagedByAuthorization"
+        },
+        "allowedThirdPartyExtensions": {
```

Comment:
do we actually return these properties today?

Reply:
No, we don't. Fixed.